workflows/proteomics/metaproteomics-gtn/README.md

```diff
@@ -0,0 +1,6 @@
+# MetaProteomics Tutorial
+In this metaproteomics worflow, we will identify expressed proteins from a complex bacterial community sample. For this MS/MS data will be matched to peptide sequences provided through a FASTA file.
```

Comment:
```suggestion
In this metaproteomics workflow, we will identify expressed proteins from a complex bacterial community sample. For this MS/MS data will be matched to peptide sequences provided through a FASTA file.
```

Reply:
A mention of the GO terms list (third input) and its expected format or where to obtain it would also be good.

Reply:
Done - see README.md